app/Jobs/Service/Install/VerifyInstallation.php

```
@@ -63,9 +63,10 @@ public function handle(): void
             // 1. Vérification des fichiers essentiels
             foreach ($verifications['files'] as $file) {
                 // Construction de la commande SSH avec Process
+                $sshPassword = config('batistack.ssh.password') ?? env('SSH_PASSWORD');
                 $sshCommand = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sshCommand'. (undefined)

(UnusedLocalVariable)